### PR TITLE
fix speedbook crash

### PR DESCRIPTION
**Pull request by DakkJaniels**
Fixes speed book crash introduced under #5674

### CODE REVIEW

File: Source/panels/spell_list.cpp [hunk 1]
```diff
@@ -339,6 +339,8 @@ void DoSpeedBook()
 			case SpellType::Charges:
 				spells = myPlayer._pISpells;
 				break;
+			default:
+				spells = 0;
```
Comment (StephenCWills) [inline Source/panels/spell_list.cpp:343]:
```suggestion
			default:
				spells = 0;
				break;
```

**Comment by StephenCWills** [comment]
This does fix the crash, but the speedbook does still have a problem.

image

**Review by StephenCWills — DISMISSED** [review]

**Review by StephenCWills — DISMISSED** [review]

**Comment by AJenbo** [comment]
With https://github.com/diasurgical/devilutionX/pull/5732 merged should we close this?

**Comment by StephenCWills** [comment]
With that one merged, this one is unnecessary. However, it's not a bad thing to have valid default cases for these switch statements.

**Comment by StephenCWills** [comment]
I screwed up a merge and ended up squashing to fix it.

**Review by StephenCWills — APPROVED** [review]

**Pull request merged**